README.md

```diff
@@ -69,7 +69,7 @@ export AWS_SECRET_ACCESS_KEY=<wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY>
 
 Start the Tile Server using
 ```shell
-docker compose up -d
+docker compose -f docker/docker-compose.yml up -d
```

Comment:
Glad you moved the docker files around! Ty!

Reply:
No problem! Docker files and docker compose had been moved but I did update the path to match the prior change.